clients/privacy-center/types/config.ts

```diff
@@ -31,4 +32,5 @@ export type ConfigConsentOption = {
   highlight?: boolean;
   name: string;
   url: string;
+  executable?: boolean;
```

Comment:
Please document these new configurations in the Readme section: https://github.com/ethyca/fides/blob/0ec57e730bd87802ffed6142e92e32df70b6533e/clients/privacy-center/README.md#L4-L5

Reply:
@allisonking can you tackle this one?

Reply:
yup yup